src/index.html

```diff
@@ -13,20 +13,32 @@
     <title>Document</title>
     <link
       rel="stylesheet"
-      href="style.css"
+      href="style.scss"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      data-qa="big"
+      class="form"
+    >
+      <input
+        data-qa="keypress"
+        type="text"
+        placeholder="Try “Los Angeles“"
+        class="form__input form__input--big"
```

Comment:
The class names 'form__input--big' and 'form__input--small' are not very descriptive. It's better to use class names that describe the content or functionality of the elements they are applied to, not their appearance. For example, you could use 'form__input--primary' and 'form__input--secondary'.